code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -1,16 +1,18 @@
 /obj/item/projectile/energy/bolt //ebow bolts
 	name = "bolt"
 	icon_state = "cbbolt"
-	damage = 8
+	damage = 15
 	damage_type = TOX
 	nodamage = 0
 	knockdown = 100
 	stutter = 5
-	drowsy = 50
+	drowsy = 30
 
 /obj/item/projectile/energy/bolt/halloween
 	name = "candy corn"
 	icon_state = "candy_corn"
 
 /obj/item/projectile/energy/bolt/large
 	damage = 20
+	knockdown = 79
+	drowsy = 0

```

Comment:
There is no shame giving it 10-15 points, it's already pretty low DPS and doesn't efficiently knock down anymore.